src/CONST.ts

```diff
@@ -4072,6 +4072,24 @@ const CONST = {
     SESSION_STORAGE_KEYS: {
         INITIAL_URL: 'INITIAL_URL',
     },
+    DEFAULT_TAX: {
+        RATES: {
```

Comment:
Why are the default values nested in `RATES`? Can't we just have them be at the root of the `DEFAULT_TAX` object?